fold-this.el

```diff
@@ -348,7 +348,10 @@ Drop the tail of the alist."
 (define-minor-mode fold-this-mode
   "Toggle folding on or off.
 With folding activated add custom map \\[fold-this-keymap]"
-  :lighter (:eval (concat " " fold-this-overlay-text))
+  :lighter (:eval (apply 'concat " "
+			 (if (listp fold-this-overlay-text)
+			     fold-this-overlay-text
+			   (list fold-this-overlay-text))))
```

Comment:
Can we fix the whitespace here? Use spaces instead of tabs.

Reply:
Yes, I'll fix this

Reply:
By the way, does anyone see a better way to do the check. I find the "if" test form ugly.

Reply:
in dash there is `(-list)` function which basically does this.